src/views/AlumniUpdateForm/index.js

```diff
@@ -0,0 +1,21 @@
+import GordonLoader from 'components/Loader';
+import GordonUnauthorized from 'components/GordonUnauthorized';
+import { useUser } from 'hooks';
+import UpdateForm from './components/UpdateForm';
+
+/**
+ * A form for alumni to request an update to their profile information.
+ */
+
+const AlumniUpdateForm = (props) => {
```

Comment:
Props aren't used in this component
```suggestion
const AlumniUpdateForm = () => {
```